src/service/video-manager-impl.js

```diff
@@ -355,6 +355,36 @@ export class VideoManager {
   }
 
   /**
+   * Returns the entry in the video manager corresponding to the element
+   * provided
+   *
+   * @param {!AmpElement} element
+   * @return {VideoEntry} entry
+   * @private
+   */
+  getEntryForElement_(element) {
+    for (let i = 0; i < this.entries_.length; i++) {
+      const entry = this.entries_[i];
+      if (entry.video.element === element) {
+        return entry;
+      }
+    }
+    dev().assert(false, 'video is not registered to this video manager');
```

Comment:
Will do. The logging has helped me while developing so I'll keep it in with the assert changed to an error